.Rbuildignore

```diff
@@ -11,7 +11,9 @@
 ^codecov\.yml$
 ^\.httr-oauth$
 ^_pkgdown\.yml$
+^doc$
```

Comment:
Why do you add these?

Reply:
To ignore the output generated by `devtools::build_vignettes()`.

Reply:
Ah, that's a workflow that we no longer recommend.

Reply:
So, you want me to remove the added ignore rules?